websites/api-docs-smoke-test/source-raml/test/api.raml

```diff
@@ -72,29 +72,29 @@ uses:
       displayName: Get all resources
       description: Query/retrieve (GET) a specific resource or resource(s). This should return a paged query result of _blah_.
       queryParameters:
-        where:
+        stringParameter:
           type: string
           required: false
           example: 6 in arrayOfIntExample
-        sort:
-          type: string
-          required: false
-          example: DESC
-        expand:
-          type: string
-          required: false
-          example: arrayOfObjectType[*]
-        limit:
+        floatParameter:
           type: number
           required: false
           example: 50
-        offset:
+        integerParameter:
+          type: integer
+          required: false
+          example: 50
+        withMarkdownDescription:
           type: number
+          description: This `markdown` should be parsed correctly.
           required: false
-          example: 5
-        collection:
+          example: 50
+        objectParameter:
           type: objects.ObjectTestType
           required: false
+        arrayOfObject:
```

Comment:
array of object is a very unrealistic, in fact technically impossible use case for a query parameter.  Since individual parameters can only be primitives,  sending the same HTTP query parameter multiple times can only be an array of String / Number / boolean,  And boolean does not really make sense to send multiple times, too. 

I suggest to rather test an Array of Stirngs.